src/dispatch/incident_cost_model/views.py

```diff
@@ -0,0 +1,103 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+import logging
+from starlette.requests import Request
+from sqlalchemy.exc import IntegrityError
+from typing import Annotated
+
+from dispatch.auth.permissions import SensitiveProjectActionPermission, PermissionsDependency
+from dispatch.database.core import DbSession
+from dispatch.database.service import CommonParameters, search_filter_sort_paginate
+from dispatch.models import PrimaryKey
+
+from .models import (
+    IncidentCostModel,
+    IncidentCostModelRead,
+    IncidentCostModelCreate,
+    IncidentCostModelUpdate,
+    IncidentCostModelPagination,
+)
+from .service import create, update, delete, get_cost_model_by_id
+
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+def get_current_incident_cost_model(db_session: DbSession, request: Request) -> IncidentCostModel:
```

Comment:
What are you using to differentiate current vs not?

Reply:
Removed this function since it wasn't needed.